ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -560,7 +559,8 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
       for {
         (alice, headers) <- fixture.getUniquePartyAndAuthHeaders("Alice")
         (bob, _) <- fixture.getUniquePartyAndAuthHeaders("Bob")
-        create: domain.CreateCommand[v.Record, OptionalPkg] = iouCreateCommand(alice)
+        create: domain.CreateCommand[v.Record, domain.ContractTypeId.Template.OptionalPkg] =
+          iouCreateCommand(alice)
```

Comment:
And so on; it seems silly to annotate the return type of `iouCreateCommand` _every single time_.